lib/model/model.go

```diff
@@ -334,8 +342,17 @@ func (m *model) addAndStartFolderLockedWithIgnores(cfg config.FolderConfiguratio
 
 	ffs := fset.MtimeFS()
 
+	if cfg.Type == config.FolderTypeEncrypted {
+		if encToken, err := fs.ReadFile(ffs, encTokenPath(cfg)); err == nil {
```

Comment:
Can we make this a structured file (json?) and include the folder ID in it?

```
{  "folderID": "foo", "token": "..." }
```

Currently it's a weakness that if everything comes crashing down and I need to offline-decrypt the files I need to know both the password and the folder ID, but the folder ID isn't stored anywhere and is probably randomly generated.